### ISSUE TITLE
Bluetooth Address is not working in iOS 12.1

### ISSUE THREAD

**Issue by Jasmeetngtech** [michaeldorner/BeeTee#23]
Bluetooth Address does not work in iOS 12.1 
**localAddress** under **BluetoothManager.h** used for finding Bluetooth Address does not work anymore in iOS 12.1, although it works in lower versions.

Any other option to find Bluetooth Address?

**Comment by Jasmeetngtech** [comment]
Any idea guys?

**Comment by michaeldorner** [comment]
Thanks for reporting. What do you mean by "not working"? Is there an exception? In the [iOS 12 Headers](https://github.com/nst/iOS-Runtime-Headers/blob/master/PrivateFrameworks/BluetoothManager.framework/BluetoothManager.h),  there is still the method 

    - (id)localAddress;

**Comment by Jasmeetngtech** [comment]
Yes, its there but it returns nil value in iOS 12.1 worked fine until 12.0.1. 
Any idea how can we find any unique device Id like IMEI, Wifi MAC address, Bluetooth Address or Serial number.
This change in iOS 12.1, actually broke my current application where we were using Bluetooth Address as Unique ID.

**Comment by Jasmeetngtech** [comment]
Getting this message" Insufficient entitlements to invoke BTLocalDeviceGetAddressString "

If this may help, or raise any ideas? Will appreciate that.
